packages/@lwc/ssr-runtime/src/reflection.ts

```diff
@@ -34,27 +104,33 @@ export function reflectAttrToProp(
     }
 }
 
-export const descriptors = create(null);
-for (const [attrName, propName] of entries(attrsToProps)) {
-    descriptors[propName] = {
-        get(this: LightningElement): string | null {
-            return this.getAttribute(attrName);
+export const descriptors: Record<string, PropertyDescriptor> = {
+    accessKey: stringDescriptor('accesskey'),
+    dir: stringDescriptor('dir'),
+    draggable: booleanDescriptor('draggable', true),
+    hidden: booleanDescriptor('hidden', true),
+    id: stringDescriptor('id'),
+    lang: stringDescriptor('lang'),
+    spellcheck: booleanDescriptor('spellcheck', false),
```

Comment:
`html-attributes.ts` in `@lwc/shared` also has a list of boolean attributes.

Reply:
That's not super relevant, because this file is global HTML attributes and none of the boolean attributes are global (except `hidden`, which I've implemented incorrectly 😓).